docs/source/api_reference/models.rst

```diff
@@ -393,6 +393,21 @@ Note that this is an alternative implementation for MPNN and currently you can o
 .. autoclass:: deepchem.models.torch_models.MPNNModel
   :members:
 
+InfoGraphModel
+--------------
+.. autoclass:: deepchem.models.torch_models.InfoGraphModel
+  :members:
+
+InfoGraphEncoder
+----------------
```

Comment:
We want these to go with the other layers since they aren't models and instead are nn.layers

Reply:
Quick reminder on this TODO